app/gateway/2.6.x/developer-portal/configuration/authentication/oidc.md

```diff
@@ -15,15 +15,19 @@ In addition, a configuration object is required to enable OIDC. Refer to the
 [Sample Configuration Object](#/sample-configuration-object) section of this
 document for more information.
 
+If you want to use the same email address for both your Kong Manager and Dev Portal accounts, ensure the Admin `username` is not set to the account email and that instead, the `custom_id` is set to the account email.
+
 **Note:** The Dev Portal does not automatically create developer accounts on login via OIDC.
 A developer account matching the `consumer_claim` configuration parameter has to be
 created and approved (if auto approve is not enabled) beforehand.
 
 OIDC for the Dev Portal can be enabled in one of the following ways:
 
-- [Kong Manager](#enable-oidc-using-kong-manager)
-- [Kong Admin API (command line)](#enable-oidc-using-the-command-line)
-- [The Kong configuration file](#enable-oidc-using-kongconf)
+- [Portal Session Plugin Config](#portal-session-plugin-config)
+- [Sample Configuration Object](#sample-configuration-object)
+- [Enable OIDC using Kong Manager](#enable-oidc-using-kong-manager)
+- [Enable OIDC using the Command Line](#enable-oidc-using-the-command-line)
+- [Enable OIDC using kong.conf](#enable-oidc-using-kongconf)
```

Comment:
```suggestion
- [Using Kong Manager](#enable-oidc-using-kong-manager)
- [Using the Command Line](#enable-oidc-using-the-command-line)
- [Using kong.conf](#enable-oidc-using-kongconf)
```

I wouldn't add those two links, as they aren't config methods. 

This topic just needs way more help - the Portal Session Plugin Config section is pointless, and the Sample Configuration Object section is out of context - it's a sample codeblock to use while configuring with any of the listed methods, but that's not very clear from the way this topic is laid out. 

All that aside - what I'm talking about are audit edits, and you shouldn't address them right now, but that's why you shouldn't add links to the sections 😄 .

Reply:
I didn't add these, I'm not sure why they are showing up like this.

Reply:
Please delete as necessary.